packages/core/features/src/index.ts

```diff
@@ -9,7 +9,7 @@ import type { StandardEventsFeature } from './events.js';
  *
  * @group Features
  */
-export type StandardFeatures = StandardConnectFeature | StandardDisconnectFeature | StandardEventsFeature;
+export type StandardFeatures = StandardConnectFeature & StandardDisconnectFeature & StandardEventsFeature;
```

Comment:
Hmm this would be a breaking change. I think it also doesn't match the intent, which was to express "this is the set of all `standard:` features". No wallet is required to implement all standard features.

However, in retrospect I think this type is pretty useless in either case. What's really the point of describing the set of standard features if they are all optional?

A wallet implementing this can still convert this to an intersection, or just declare the individual standard features they support as an intersection since that's still a subset of the set of all standard features.

Reply:
I think if you were guaranteed to use all of them, you'd do `WalletWithFeatures<UnionToIntersection<StandardFeatures>>`.

The point of the union therefore is to express "these are the entire set of features that _may_ be supported by the wallet" but since the features can change dynamically, there's no guarantee that all of them will be supported at a given time.

In practice, I'm not sure this even matters except to a wallet, because the types of features of any wallet are not known by our code or an app's, and actual features must be detected at runtime.